models/flight-model.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+const AutoIncrement = require('mongoose-sequence')(mongoose);
+
+const flightSchema = mongoose.Schema({
+  seatPlan: {
+    type: String,
```

Comment:
are you sure it's a string? what would be its value? I think a flight has N rows of 6 seats each, and you should track if any of the seats are empty or reserved, and their prices...